docs/source/install.rst

```diff
@@ -200,6 +200,47 @@ The test suite requires that the proj-datumgrid package is installed in
 :envvar:`PROJ_LIB`.
 
 
+Autotoos configure options
+++++++++++++++++++++++++++
+
+Most POSIX systems may not require any options to ``./configure`` if all
+PROJ requirements are met, installed into common directories, and a
+"default" behaviour is desired.
+
+Some influential environment variables are used by ``./configure``,
+with no expected defaults:
+
+.. envvar:: CC
+
+    C compiler command.
+
+.. envvar:: CFLAGS
+
+    C compiler flags.
+
+.. envvar:: CXX
+
+    C++ compiler command.
+
+.. envvar:: CXXFLAGS
+
+    C++ compiler flags
+
+See ``./configure --help`` for all options, here are a few key options:
+
+.. option:: --enable-lto
+
+    Enable compiler's Link Time Optimization, default disabled.
+
+.. option:: --disable-tiff
```

Comment:
Should also include `--with-curl`  here

Reply:
Added, along with CMake's `CURL_INCLUDE_DIR` and `CURL_LIBRARY` options.